build.gradle

```diff
@@ -54,9 +57,13 @@ afterEvaluate {
     }
 }
 
+
 allprojects {
-    group = 'org.spekframework.spek2'
+//    group = 'org.spekframework.spek2'
```

Comment:
?

Reply:
+1

Reply:
It breaks the build, that's why I commented it out. I'll add a comment. It causes a circular dependency, it somehow changes the gradle project path.